client/stpIfPol_service.go

```diff
@@ -0,0 +1,53 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/models"
+)
+
+func (sm *ServiceManager) CreateSpanningTreeInterfacePolicy(name string, description string, stpIfPolattr models.SpanningTreeInterfacePolicyAttributes) (*models.SpanningTreeInterfacePolicy, error) {
+	rn := fmt.Sprintf("infra/ifPol-%s", name)
```

Comment:
I am not sure if this is correct as rn should probably only be ifPol-%s and parentDn should be uni/infra

Reply:
Also should we move the rn and parentDn into the model definition and just refer to it here (and in the other functions below)?

Reply:
Why would we want to move rn and parentDn though?

Reply:
Done